assets/src/blocks/CarouselHeader/SidebarSlide.js

```diff
@@ -23,45 +36,104 @@ export const SidebarSlide = ({
     changeSlideAttribute('focal_points', index)(focalPoints);
   }
 
+  const onToggleHandler = (opened) => {
+    setIsOpened(opened);
+  }
+
+  useEffect(() => {
+    if(isOpened) {
+      goToSlideHandler(index);
+    }
+  }, [ isOpened ]);
+
   return (
-    <>
-      <div className='carousel-header-slide-container'>
-        <div className='carousel-header-slide-options-wrapper'>
-          <div className='row'>
-            <div className='col'>
-              <URLInput
-                label={__('Url for link', 'planet4-blocks-backend')}
-                value={link_url}
-                onChange={changeSlideAttribute('link_url', index)}
-              />
+    <div
+      className={`sidebar-slide ${isOpened ? 'opened-slide' : ''}`}
+      data-index={index}
+      draggable={!isOpened}
+      onDragStart={onDragStartHandler}
+      onDragEnd={onDragEndHandler}
+      onDragOver={onDragOverHandler}
+      onDragLeave={onDragLeaveHandler}
+    >
+      <PanelBody
+        key={ index }
+        initialOpen={ isOpened }
+        onToggle={onToggleHandler}
+        title={
+          <div className='slide-item-wrapper'>
+            <div
+              className='slide-item-draggable-button'
+              onClick={(evt) => {
+                // This method avoids to propagate the toggle event
+                evt.stopPropagation();
+              }}
+            >
+              <svg width="8" height="14" viewBox="0 0 8 14" fill="none" xmlns="http://www.w3.org/2000/svg"><rect width="2" height="2" fill="black"/><rect x="6" width="2" height="2" fill="black"/><rect y="6" width="2" height="2" fill="black"/><rect x="6" y="6" width="2" height="2" fill="black"/><rect y="12" width="2" height="2" fill="black"/><rect x="6" y="12" width="2" height="2" fill="black"/></svg>
             </div>
-          </div>
-          <div className='row'>
-            <div className='col'>
-              <CheckboxControl
-                label={__('Open in a new tab', 'planet4-blocks-backend')}
-                value={link_url_new_tab}
-                checked={link_url_new_tab}
-                onChange={changeSlideAttribute('link_url_new_tab', index)}
-              />
+            <div className='slide-item-order-arrows'>
+              <div
```

Comment:
For accessibility we should make this a button. Else you cannot focus it with the keyboard, nor press it.

Then we can also put a `disabled` attribute on the button and don't need the class `disabled`.

Reply:
I wanted to use a `Button` but I figured out that, because of the Accessibility, you cannot nest interactive elements.

In other words, you cannot do this:
```
<button>
   <button></button>
</button>
```

As you see [here](https://github.com/WordPress/gutenberg/blob/7bf1297d3751dbb959e62c32666321a3af9a03dc/packages/components/src/panel/body.js#L100), the Gutenberg's PanelBody component is developed

Reply:
We could also use something other than the `PanelBody` component as a container. Since it's a button, it's clearly not intended to put stuff in (my bad I didn't spot this earlier).

Reply:
We can also do this in a follow up, since avoiding `PanelBody` is not a quick fix, and without the PR there's no buttons at all. But since we're adding the button as the only option for keyboard users, we should make it a button element so that they can definitely reach it.

We could also add a tabindex attribute to this, which might make it focusable. But that's also potentially tricky in React.